vlib/v/checker/tests/go_wait_or.out

```diff
@@ -19,7 +19,7 @@ vlib/v/checker/tests/go_wait_or.vv:19:13: error: unexpected `or` block, the func
       |                ~~~~~~~~~~~~~~~~~~~~~~~
    20 |     tg2[0].wait()?
    21 |     tg3 := [
-vlib/v/checker/tests/go_wait_or.vv:20:15: error: unexpected `?`, the function `wait` does not return an Option or a Result
```

Comment:
The error handler explicitly checks if we `.propagate_option` or `.propagate_result`.

I assume we therefore don't need the `or a Result`. Other errors already have taken that into account end ended with just `an Option`. It was just this error that went out of the line.